.travis.yml

```diff
@@ -323,7 +334,11 @@ script:
 
   # compile and execute unit tests
   - mkdir -p build && cd build
-  - cmake .. ${CMAKE_OPTIONS} -DJSON_MultipleHeaders=${MULTIPLE_HEADERS} -DJSON_BuildTests=On -GNinja && cmake --build . --config Release
+  - if [[ "${CMAKE_CXX_STANDARD}" != "" ]]; then
+    cmake .. ${CMAKE_OPTIONS} -DJSON_MultipleHeaders=${MULTIPLE_HEADERS} -DCMAKE_CXX_STANDARD=${CMAKE_CXX_STANDARD} -DJSON_BuildTests=On -GNinja && cmake --build . --config Release ;
```

Comment:
I think it may be clearer to set `CMAKE_CXX_STANDARD` by default to `11` (if I understand the Travis YAML correctly, this needs to be done at the top of the file) and only override it to `17` where a different value is needed. This avoids this `if` here, and also makes it easier to add similar tests for `14` and `20`.